src/hq.ini

```diff
@@ -0,0 +1,8 @@
+# Fill this out with your user and device information and rename this to hq.ini
+
+[Main]
+bearer_token = eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjE2Mjg3MTI3LCJ1c2VybmFtZSI6IlRoYXREaXJ0eUQiLCJhdmF0YXJVcmwiOiJzMzovL2h5cGVzcGFjZS1xdWl6L2RlZmF1bHRfYXZhdGFycy9VbnRpdGxlZC0xXzAwMDNfcmVkLnBuZyIsInRva2VuIjoiaXJqQVRWIiwicm9sZXMiOltdLCJjbGllbnQiOiIiLCJndWVzdElkIjpudWxsLCJ2IjoxLCJpYXQiOjE1MjcwNDQ4NDksImV4cCI6MTUzNDgyMDg0OSwiaXNzIjoiaHlwZXF1aXovMSJ9.XqQWdzzLy5LkeBCIRfdqT1iineaG_mxB2XejRvFwUWk
```

Comment:
Same here with the bearer_token, although I'm not sure if you can even get a new one. I'll get in touch with how to remove it from git